CHANGELOG.md

```diff
@@ -15,9 +15,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Improve "Handle Device Permission" guide in Quick Starts.
+
 ### Fixed
 
 - Fix command line sample to install latest node dependencies in migration guides.
+- Fix the broken markdown URLs due to this [storybook issue](https://github.com/storybookjs/storybook/issues/9005) by replacing markdown links with HTML `<a>` tag.
```

Comment:
So our issue is more like https://github.com/storybookjs/storybook/issues/17308 and not the absolute URLs right?

Reply:
Same thing, and this workaround is from 9005.

![image](https://user-images.githubusercontent.com/20471438/163651048-df25e2d1-396a-459c-9878-0065e942b712.png)

https://github.com/storybookjs/storybook/issues/9005#issuecomment-567885987

Reply:
Got it thanks for checking and explaining!